doc/devices/dts/device_tree.rst

```diff
@@ -60,11 +60,11 @@ already have defined for a given platform.
 Today, configuration in Zephyr comes from a number of different places.  It can
 come from Kconfig files, CMSIS header files, vendor header files, prj.conf
 files, and other miscellaneous sources.  The intent of using device tree is to
-replace or curtail the use of Kconfig files throughout the system, and instead
-use device tree files to describe the configuration of device nodes.  CMSIS and
-vendor header files can be used in conjunction with the device tree to fully
-describe hardware.  Device tree is not intended to replace CMSIS or vendor
-include files.
+replace or curtail the use of some Kconfig files throughout the system, and
+instead use device tree files to describe the configuration of device nodes.
```

Comment:
"replace or curtail the use of some Kconfig files throughout the system, and instead use device tree files to describe the configuration of device nodes"
Isn't it rather a temporary side effect of device tree introduction?
While it makes sense to us, this might be unclear for a future Zephyr user.
I think we'd rather remove this part or provide a generic description with no reference with Kconfig, as:
"Device tree provides a unified description of a hardware system used in an application. It is used in Zephyr to describe hardware and provide a boot time configuration."

Reply:
Additionnaly, we might want to add a reference to specification, if not already present  (http://www.devicetre.org/)